src/components/SettingMenu/SettingMenu.tsx

```diff
@@ -63,18 +65,26 @@ const SettingMenu = ({ showLogout, logout }: SettingMenuProps) => {
           <MenuItemAnchor
             title="Dashboard"
             link="https://dashboard.etherspot.io"
-            icon={<MdOutlineDashboardCustomize size={16} style={{ marginRight: '12px' }} />}
+            icon={<MdOutlineDashboardCustomize size={16} style={{ marginRight: '12' }} />}
```

Comment:
Sorry I don't follow why u removed px?
```suggestion
            icon={<MdOutlineDashboardCustomize size={16} style={{ marginRight: '12px' }} />}
```

Reply:
It was in the PR comment. Will change it.

Reply:
It was supposed to just be the number @parag-chauhan, without the quotation marks

Reply:
> It was supposed to just be the number @parag-chauhan, without the quotation marks `marginRight: 12`

oh okay! my bad @Yash-Chaubal . But i guess its okay to keep the 'px',  been used for rest of the icons as well.